### PR TITLE
{monitor}Fix doc issues of monitor diagnostic-settings

### PR DESCRIPTION
**Pull request by YuanyuanNi**
**Description**

Fix doc issues of monitor diagnostic-settings
**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 1]
```diff
@@ -849,10 +843,10 @@
   - name: --event-hub-rule
     short-summary: Name or ID of the event hub authorization rule.
 examples:
-  - name: Create diagnostic settings with EventHub.
+  - name: Create diagnostic settings , retention here only applies when the target is a storage account.
```
Comment (necusjz) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py:846]:
```suggestion
  - name: Create diagnostic settings, retention here only applies when the target is a storage account.
```

Comment (necusjz) [reply src/azure-cli/azure/cli/command_modules/monitor/_help.py:846]: a mirror issue here : )

**Comment by yonzhan** [comment]
monitor

File: src/azure-cli/azure/cli/command_modules/monitor/_params.py [hunk 2]
```diff
@@ -283,8 +283,10 @@ def load_arguments(self, _):
 
     with self.argument_context('monitor diagnostic-settings create') as c:
         c.resource_parameter('resource_uri', required=True, arg_group='Target Resource', skip_validator=True)
-        c.argument('logs', type=get_json_object)
-        c.argument('metrics', type=get_json_object)
+        c.argument('logs', type=get_json_object, help=" JSON encoded list of logs settings. Use '@{file}' to load from a file."
```
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/monitor/_params.py:286]:
```suggestion
        c.argument('logs', type=get_json_object, help="JSON encoded list of logs settings. Use '@{file}' to load from a file."
```

**Review by kairu-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**